language-sql.md

```diff
@@ -3,11 +3,33 @@
 ### Minimum Requirements
 
 - MySQL keywords in uppercase
-- Column names in lowercase
+    E.g.: 
+    ```
+    UPDATE `consumer` SET `PASSWORD` = MD5(`PASSWORD`);
+    ```
+
+- Database table names must be plural. 
+    E.g.: 
+    `users`, `subscriptions`
+    
+    for Linking tables :
+    `users_subscriptions`
```

Comment:
@piadelosreyes yes
also, i think the naming will also be dependent on the relationship of the tables.
to better understand what i meant:

`one-to-many` `<singular>_<plural>` table name
`many-to-many` `<plural>_<plural>`  table name

😄